doc/implementMyOwnNode.md

```diff
@@ -1,7 +1,30 @@
-TBD
+If none of the default implementation meet your requirements, feel
+free to implement your own. 
 
-Java
-Spring
-SPI
 
-REST API?
+## Implement your own node adapter
+
+If you can run the default node implementation in your infrastructure, but need to integrate with your own
+back end, you can implement your own adapter:
+
+- interfaces are defined in the `node-api` module
+- for an example, have a look at `adapter-example-spring` 
```

Comment:
I would also mention the excel or dummy adapter as an example for a non-spring-based adapter.